# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../deva/libmaverik.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./examples/Makefile	2002-01-16 16:18:53.000000000 +0100
+++ ./examples/Makefile-new	2004-01-02 19:47:03.000000000 +0100
@@ -39,7 +39,7 @@
 
 
 ${DBIN}/DEVA_server:	DEVA_server.o
-	${CXX} ${CXXFLAG} DEVA_server.o -o ${DBIN}/DEVA_server ${LIBDIR} -ldeva_server -ldeva_comms -ldeva_kernel -lpthread ${DYNAMIC}
+	${CXX} ${CXXFLAG} DEVA_server.o -o ${DBIN}/DEVA_server ${LIBDIR} -ldeva_server -ldeva_comms -L${MAV_HOME}/lib -lmaverik -ldeva_kernel -lpthread ${DYNAMIC}
 
 ${DBIN}/DEVA_client:	DEVA_client.o
 	${CXX} ${CXXFLAGS} DEVA_client.o -o ${DBIN}/DEVA_client ${LIBDIR} -ldeva_viewer -L${MAV_HOME}/lib -lmaverik -ldeva_client -ldeva_comms -ldeva_kernel ${DYNAMIC}
